src/pages/index.js

```diff
@@ -54,39 +72,136 @@ const profileEditModal = new ModalWithForm(
   handleProfileEditSubmit
 );
 profileEditModal.setEventListeners();
+
 const addCardModal = new ModalWithForm("#add-card-modal", handleAddCardCreate);
 addCardModal.setEventListeners();
 
+const profileAvatarEditModal = new ModalWithForm(
+  "#profile-avatar-edit-modal",
+  handleProfileAvatarUpdate
+);
+profileAvatarEditModal.setEventListeners();
+
+// Add an event listener to the button to open the avatar update modal
+const profileAvatarEditButton = document.querySelector(
+  "#profile-avatar-edit-button"
+);
+profileAvatarEditButton.addEventListener("click", () => {
+  profileAvatarEditModal.open();
+  profileAvatarInput.value = "";
+});
+
 /*    Form validators    */
-console.log(document.forms);
 const profileEditForm = document.forms["profile-edit-form"];
 const addCardForm = document.forms["add-card-form"];
+const profileAvatarForm = document.forms["profile-avatar-edit-form"];
 const editFormValidator = new FormValidator(config, profileEditForm);
 const addFormValidator = new FormValidator(config, addCardForm);
+const avatarFormValidator = new FormValidator(config, profileAvatarForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object: `formValidators`.  Then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // Here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // Here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```